src/service/extension-location.js

```diff
@@ -45,15 +45,6 @@ export function calculateScriptBaseUrl(location, opt_isLocalDev) {
   return urls.cdn;
 }
 
-/**
- * Calculates if we need a single pass folder or not.
- *
- * @return {string}
- */
-function getSinglePassExperimentPath() {
-  return getMode().singlePassType ? `${getMode().singlePassType}/` : '';
```

Comment:
Can we remove `getMode().singlePassType`, too?

Reply:
will do

Reply:
I see references to `singlePassType` across 7 files: https://github.com/ampproject/amphtml/search?q=singlePassType&unscoped_q=singlePassType

Do the others need to be removed?

Reply:
hmm looking back at the other places where we have it, it is actually needed for the a4a team since that's the only way they are able to slice/filter the experiment on their monitoring system i'll need to leave it in

Reply:
ive reverted the removal and fixed the url (and its tests) pathing to just remove the `${spPath}`